app/controllers/doorkeeper/tokens_controller.rb

```diff
@@ -37,5 +37,9 @@ def revoke_token(token)
     def strategy
       @strategy ||= server.token_request params[:grant_type]
     end
+
+    def authorize_response
+      @authorize_response ||= strategy.authorize
```

Comment:
Awesome, going to update this to make it more explicit, and add the CHANGELOG entry, that should be ready by tomorrow.

Thanks!